domains/akhilesh.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Akhileshrangani4",
+        "email": "akhileshrangani4@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=jr4pbfhu2ujqjmfocyc-ukbxokjugvn8oit17d-qhia"
```

Comment:
Google site verification through TXT records is not supported, use the HTTPS method